scripts/local_k8s/deployment.py

```
@@ -2,39 +2,68 @@
+import secrets
+from collections.abc import Callable
+
 from .commands import run
 from .config import PreviewConfig
-from .k3d import ensure_cluster, import_image, print_cluster_status
-from .k8s import ensure_namespace, print_kubernetes_status
+from .cluster import ensure_cluster, import_image, print_cluster_status
+from .k8s import _helm_fullname, ensure_namespace, print_kubernetes_status
 from .validation import LocalK8sError, require_tools
 
+SESSION_SECRET_KEY_NAME = "session_key"
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to: "SESSION_SECRET_KEY_NAME"

(S105)